sql: TestRelocateVoters failed

sql.TestRelocateVoters [failed](https://tanzanite.cluster.engflow.com/invocations/default/8d5b8c67-35b6-4168-965f-87485669da8d?testReportRun=1&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvc3FsOnNxbF90ZXN0) on master @ [ff291e3ee70eedac50b985860502d2f0b0867003](https://github.com/cockroachdb/cockroach/commits/ff291e3ee70eedac50b985860502d2f0b0867003):


```
=== RUN   TestRelocateVoters
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestRelocateVoters1228001707
    test_log_scope.go:81: use -show-logs to present logs inline
    multitenant_admin_function_test.go:768: -- test log scope end --
test logs left over in: outputs.zip/logTestRelocateVoters1228001707
--- FAIL: TestRelocateVoters (760.89s)
=== RUN   TestRelocateVoters/ALTER_RANGE_RELOCATE_VOTERS
    multitenant_admin_function_test.go:377: condition failed to evaluate within 3m45s: from multitenant_admin_function_test.go:400: expected "trying to add({ChangeType:ADD_VOTER Target:n1,s1}) to a store that already has a LEARNER" contains "ok" tenant=secondary query=``ALTER RANGE RELOCATE VOTERS FROM 2 TO 1 FOR (SELECT min(range_id) FROM [SHOW RANGES FROM TABLE t]);`` leaseholder=4 replicas=[2 3 4] voting_replicas=[2 3 4] non_voting_replicas=[] fromReplica=2 toReplica=1 row=0 col=2
    --- FAIL: TestRelocateVoters/ALTER_RANGE_RELOCATE_VOTERS (511.72s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRelocateVoters.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34867

--- a/pkg/sql/multitenant_admin_function_test.go
+++ b/pkg/sql/multitenant_admin_function_test.go
@@ -683,6 +683,8 @@ func TestRelocateVoters(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	skip.UnderDuress(t, "test flakes in slow builds; see #108081")
+
 	testCases := []testCase{
 		{
 			desc:  "ALTER RANGE x RELOCATE VOTERS",
@@ -773,6 +775,8 @@ func TestRelocateVoters(t *testing.T) {
 func TestExperimentalRelocateVoters(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
+
+	skip.UnderDuress(t, "test flakes in slow builds; see #108081")
 
 	testCases := []testCase{
 		{

